Cargo.toml

```diff
@@ -1,20 +1,21 @@
 [package]
 name = "one-of-futures"
-version = "0.1.1"
+version = "0.1.2"
 authors = ["Gleb Pomykalov <glebpom@gmail.com>"]
 edition = "2018"
 description = "Future-aware Either-like type with multiple variants"
 keywords = ["async", "futures", "either"]
 categories = ["asynchronous"]
 license = "MIT"
 homepage = "https://github.com/glebpom/one-of-futures"
-documentation = "https://docs.rs/one-of-futures/0.1.1/one-of-futures/"
+documentation = "https://docs.rs/one-of-futures/0.1.2/one-of-futures/"
 repository = "https://github.com/glebpom/one-of-futures"
 readme = "README.md"
 
 [dependencies]
 futures-core = "0.3.1"
```

Comment:
since we have futures 0.1 support now, future 0.3 should be also optional, enabled by default.